packages/core/src/checkout-buttons/create-checkout-button-registry.ts

```diff
@@ -117,6 +117,15 @@ export default function createCheckoutButtonRegistry(
         )
     );
 
+    registry.register(CheckoutButtonMethodType.BRAINTREE_VENMO, () =>
+        new BraintreeVenmoButtonStrategy(
+            store,
+            paymentMethodActionCreator,
+            new BraintreeSDKCreator(new BraintreeScriptLoader(scriptLoader)),
```

Comment:
Can we move this to a variable, since I can see this being the same for other strategies.

Reply:
Yes we can. It is a great idea